VideoLocker/src/main/java/org/edx/mobile/view/NavigationFragment.java

```diff
@@ -231,6 +233,7 @@ public void onClick(View v) {
             @Override
             public void onClick(View v) {
                 environment.getRouter().forceLogout(getActivity(), environment.getSegment(), environment.getNotificationDelegate());
+                SecurityUtil.clearUserData(MainApplication.application);
```

Comment:
The token invalidation was also supposed to be only performed upon manual logout. If an automatic logout is performed due to an authorization issue, then the invalidation would seem to be meaningless (though I suppose there's no harm in doing so).